build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -25,6 +25,9 @@ var _modulePartialExtension = _context.withoutExtension(jsModuleExtension);
 Future<void> bootstrapDdc(BuildStep buildStep,
     {DartPlatform platform,
     Set<String> skipPlatformCheckPackages = const {}}) async {
+  // Ensures that the sdk resources are built and available.
```

Comment:
[nit] this comment feels redundant, the method name is pretty clear

Reply:
given how slow our CI is right now I am just going to leave it - it does give some additional reasoning for why also :)